src/components/ui/input.tsx

```
@@ -0,0 +1,19 @@
+import * as React from "react"
```

ESLint

[error] Insert `;`

(prettier/prettier)